ui/src/components/FormSubmitCancel.vue

```diff
@@ -17,6 +17,11 @@
           Cancel
         </o-button>
       </div>
+      <div class="control" v-if="showClear">
```

Comment:
UI-wise, I think it would look better with an X instead of a button 'clear',  (optional)

![image](https://user-images.githubusercontent.com/361916/202450476-2088812b-0f29-4140-bcdb-5d2af2d585eb.png)

Reply:
Yes, I expected you'd say that. Unfortunately, [oruga select](https://oruga.io/components/select.html) does not appear to allow that